msal/src/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -75,31 +77,42 @@ internal AuthenticationResult(MsalAccessTokenCacheItem msalAccessTokenCacheItem,
         public virtual DateTimeOffset ExpiresOn => _msalAccessTokenCacheItem.ExpiresOn;
 
         /// <summary>
-        /// Gets an identifier for the tenant the token was acquired from. This property will be null if tenant information is
+        /// Gets an identifier for the Azure AD tenant from which the token was acquired. This property will be null if tenant information is
         /// not returned by the service.
         /// </summary>
         public virtual string TenantId => _msalIdTokenCacheItem?.IdToken?.TenantId;
 
         /// <summary>
-        /// Gets the account object. Some elements in Account might be null if not returned by the
-        /// service. It can be passed back in some API overloads to identify which account should be used.
+        /// Gets the account information. Some elements in Account might be null if not returned by the
+        /// service. The account can be passed back in some API overloads to identify which account should be used such 
+        /// as <see cref="IClientApplicationBase.AcquireTokenSilentAsync(IEnumerable{string}, IAccount)"/> or
+        /// <see cref="IClientApplicationBase.RemoveAsync(IAccount)"/>
         /// </summary>
         public virtual IAccount Account { get; internal set; }
 
         /// <summary>
-        /// Gets the entire Id Token if returned by the service or null if no Id Token is returned.
+        /// Gets the  Id Token if returned by the service or null if no Id Token is returned.
         /// </summary>
         public virtual string IdToken => _msalIdTokenCacheItem.Secret;
```

Comment:
if this is the id?token what is then ?msalIdTokenCacheItem.IdToken ?

Reply:
that's a question for @SomkaPe  ?